aten/src/ATen/native/ConvUtils.h

```diff
@@ -56,6 +56,25 @@ using slow_conv_transpose3d_backward_fn = std::tuple<at::Tensor,at::Tensor,at::T
     at::IntArrayRef, at::IntArrayRef, at::IntArrayRef, std::array<bool,3>);
 DECLARE_DISPATCH(slow_conv_transpose3d_backward_fn, slow_conv_transpose3d_backward_stub);
 
+namespace {
+  static bool cudnnv8_heuristic_mode_b = c10::utils::check_env("TORCH_CUDNN_USE_HEURISTIC_MODE_B") == true ? true : false;
```

Comment:
```suggestion
  static bool cudnnv8_heuristic_mode_b = c10::utils::check_env("TORCH_CUDNN_USE_HEURISTIC_MODE_B") == true;
```

Reply:
maybe just ```static bool cudnnv8_heuristic_mode_b = c10::utils::check_env("TORCH_CUDNN_USE_HEURISTIC_MODE_B")```? :)

Reply:
Right, maybe that, `check_env` returns optional<bool> so that should work